scrapinghub/client/items.py

```diff
@@ -37,6 +39,34 @@ class Items(_DownloadableProxyMixin, _ItemsResourceProxy):
             'size': 100000,
         }]
 
+    - retrieve items via a generator of lists. This is most useful in cases
+      where the job has a huge amount of items and it needs to be broken down
+      into chunks when consumed. This example shows a job with 3 items::
+
+        >>> gen = job.items.list_iter(chunksize=2)
+        >>> next(gen)
+        [{'name': 'Item #1'}, {'name': 'Item #2'}]
+        >>> next(gen)
+        [{'name': 'Item #3'}]
+        >>> next(gen)
+        Traceback (most recent call last):
+          File "<stdin>", line 1, in <module>
+        StopIteration
+
+    - retrieving via meth::`list_iter` also supports the `start` and `count`.
+      params. This is useful when you want to only retrieve a subset of items in
+      a job. The example below belongs to a job with 10 items::
+
+        >>> gen = job.items.list_iter(chunksize=2, start=5, size=3)
```

Comment:
```suggestion
        >>> gen = job.items.list_iter(chunksize=2, start=5, count=3)
```

Reply:
Thanks for catching @vshlapakov! I think this was when I was deciding to rename `count` as `size`. 😅